src/test/java/se/sundsvall/installation/api/InstallationResourceTest.java

```diff
@@ -1,4 +1,45 @@
 package se.sundsvall.installation.api;
 
-public class InstallationResourceTest {
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+import static org.springframework.boot.test.context.SpringBootTest.WebEnvironment.RANDOM_PORT;
+import static se.sundsvall.installation.TestUtil.createParameterMap;
+import static se.sundsvall.installation.TestUtil.createSearchParameters;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.web.reactive.server.WebTestClient;
+
+import se.sundsvall.installation.Application;
+import se.sundsvall.installation.api.model.SearchParameters;
+import se.sundsvall.installation.service.InstallationService;
+
+@SpringBootTest(classes = Application.class, webEnvironment = RANDOM_PORT)
+@ActiveProfiles("junit")
+class InstallationResourceTest {
+
+	@MockBean
+	private InstallationService installationServiceMock;
+
+	@Autowired
+	private WebTestClient webTestClient;
+
+	@Test
+	void getInstallations() {
+		final var searchParameters = createSearchParameters();
+		final var parameterObject = createParameterMap(searchParameters);
+
+		webTestClient.get().uri(uriBuilder -> uriBuilder.path("/installations")
+				.queryParams(parameterObject).build())
+			.exchange()
+			.expectStatus().isOk();
+
+		verify(installationServiceMock).getInstallations(any(SearchParameters.class));
```

Comment:
Why not use a captor here to verify that the sent in parameter values are used when calling service?
```
	@Captor
	private ArgumentCaptor<SearchParameters> searchParametersCaptor;

	void getInstallations() {
		...
		verify(installationServiceMock).getInstallations(SearchParametersCaptor.capture());

  		assertThat(searchParametersCaptor.getValue()).satisfies(bean -> {
			assertThat(bean.getCategory()).isEqualTo(searchParameters.getCategory());
			assertThat(bean.getDateFrom()).isEqualTo(searchParameters.getDateFrom());
			assertThat(bean.getFacilityId()).isEqualTo(searchParameters.getFacilityId());
			assertThat(bean.getInstalled()).isEqualTo(searchParameters.getInstalled());
			assertThat(bean.getLimit()).isEqualTo(searchParameters.getLimit());
			assertThat(bean.getPage()).isEqualTo(searchParameters.getPage());
			assertThat(bean.getSortBy()).isEqualTo(searchParameters.getSortBy());
			assertThat(bean.getSortDirection()).isEqualTo(searchParameters.getSortDirection());
		});
```
(You'll discover that your createParameterMap doesn't handle lists corrrectly)